### PR TITLE
WIP: Relationships filter dropdown valus per gender RL_86

### PR DESCRIPTION
**Pull request by amanbop**

### CODE REVIEW

File: app/javascript/Clients/clientIdPage/Relationships/EditRelationsCard.test.js [hunk 1]
```diff
@@ -158,4 +214,112 @@ describe('EditRelationsCard', () => {
...
+
+    it('should return the relationship type list containing only Male and Female gender given a male primary and female secondary client and younger client as junior member', () => {
+      expect(
+        wrapper
+          .instance()
+          .relationshipTypeList(clientHarmoinefFormatted, clientFrodoFromatted)
+      ).toEqual(expect.arrayContaining([genderCodeFm]));
+    });
+
+    // it('should return the relationship type list given a primary and secondary client', () => {
```
Comment (karthik-sista) [inline app/javascript/Clients/clientIdPage/Relationships/EditRelationsCard.test.js]: Please delete the comments.

Comment (amanbop) [reply app/javascript/Clients/clientIdPage/Relationships/EditRelationsCard.test.js]: Good catch Kartik. Done

File: app/javascript/Clients/clientIdPage/Relationships/EditRelationsCard.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+// import moment from 'moment';
```
Comment (karthik-sista) [inline app/javascript/Clients/clientIdPage/Relationships/EditRelationsCard.js]: Delete this Comment too .

**Comment by karthik-sista** [comment]
The code works fine . 👍

File: app/javascript/Clients/clientIdPage/Relationships/EditRelationsCard.js (hunk 2 above)
Comment (amanbop) [reply app/javascript/Clients/clientIdPage/Relationships/EditRelationsCard.js]: Deleted

**Pull request merged**